### PR TITLE
brew: update page; brew-*: add page; brew-cask: remove page

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

I removed the page for `brew cask` because now it's just an argument that can be passed to certain subcommands (and this should be documented in pages for the respective subcommands).

### CODE REVIEW

File: pages/common/brew-install.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# brew install
+
+> Install a Homebrew formula or cask.
+> More information: <https://docs.brew.sh/Manpage#install-options-formulacask->.
+
+- Install a formula/cask:
+
+`brew install {{formula|cask}}`
```
Comment (gutjuri) [inline pages/common/brew-install.md:8]:
```suggestion
`brew install {{formula_or_cask_name}}`
```

Using the `|` syntax would imply that only the strings `formula` or `cask` are accepted.

File: pages/common/brew-install.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Build and install a formula from source (dependencies will still be installed from bottles):
+
+`brew install --build-from-source {{formula}}`
+
+- Download the manifest, print what would be installed but don't actually install anything:
+
+`brew install --dry-run {{formula|cask}}`
```
Comment (gutjuri) [inline pages/common/brew-install.md:16]:
```suggestion
`brew install --dry-run {{formula_or_cask_name}}`
```

same here

File: pages/common/brew-uninstall.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# brew uninstall
+
+> Uninstall a Homebrew formula/cask.
+> Use `brew autoremove` to remove unused dependencies afterwards.
+> More information: <https://docs.brew.sh/Manpage#uninstall-remove-rm-options-installed_formulainstalled_cask->.
+
+- Uninstall a formula/cask:
+
+`brew uninstall {{formula|cask}}`
```
Comment (gutjuri) [inline pages/common/brew-uninstall.md:9]:
```suggestion
`brew uninstall {{formula_or_cask_name}}`
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I've added some comments. This touches on a larger issue, which you can deduct from my comments.

File: pages/common/brew-install.md (hunk 1 above)
Comment (acuteenvy) [reply pages/common/brew-install.md:8]: There are a lot of pages using `|` to denote different kinds of placeholders that do not mean the only thing you can put there is the literal string used in the placeholder.
For example: `docker service inspect {{service_name|ID}}`.

In a page documenting a package manager it is quite obvious that there is more than one package to be installed.

Comment (kbdharun) [reply pages/common/brew-install.md:8]: I agree with @acuteenvy here, we use the same (`|`) on other pages to denote multiple possible options. If we go with the new proposed option with the snake case then the placeholder will become long and would complicate the example.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**